src/main/java/com/google/flourbot/api/CardResponse.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Creates card response for bot
+ */
+
+package com.google.flourbot.api;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.api.services.chat.v1.model.ActionParameter;
+import com.google.api.services.chat.v1.model.Button;
+import com.google.api.services.chat.v1.model.Card;
+import com.google.api.services.chat.v1.model.CardHeader;
+import com.google.api.services.chat.v1.model.FormAction;
+import com.google.api.services.chat.v1.model.Image;
+import com.google.api.services.chat.v1.model.ImageButton;
+import com.google.api.services.chat.v1.model.KeyValue;
+import com.google.api.services.chat.v1.model.Message;
+import com.google.api.services.chat.v1.model.OnClick;
+import com.google.api.services.chat.v1.model.OpenLink;
+import com.google.api.services.chat.v1.model.Section;
+import com.google.api.services.chat.v1.model.TextButton;
+import com.google.api.services.chat.v1.model.TextParagraph;
+import com.google.api.services.chat.v1.model.WidgetMarkup;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.logging.Logger;
+import java.util.stream.Stream;
+
+public class CardResponse {
+
+  private static final String INTERACTIVE_TEXT_BUTTON_ACTION = "doTextButtonAction";
+  private static final String INTERACTIVE_IMAGE_BUTTON_ACTION = "doImageButtonAction";
+  private static final String INTERACTIVE_BUTTON_KEY = "originalMessage";
+  private static final String HEADER_IMAGE = "https://shop.googlemerchandisestore.com/store/20160512512/assets/items/largeimages/GGOEACBA116699.jpg";
+  private static final String BOT_NAME = "MacroBot";
+  private static final String REDIRECT_URL = "https://goo.gl/kwhSNz"; // TODO: Change to spreadsheet link
+
+  /**
+    * Creates a card-formatted response based on the message sent in Hangouts Chat.
```

Comment:
Nice javadoc, which makes me think this was copied from somewhere - that's prefectly fine, but it would be nice to call it out for our internal knowledge.

Reply:
What does "call it our in our internal knowledge" mean in practice? I mentioned it was from the sample code Sydney sent in the commit message - but where should I put it next time?